package.json

```diff
@@ -60,16 +60,22 @@
     "git-branch": "^0.3.0",
     "glob": "^6.0.4",
     "glob-stream": "^6.1.0",
+    "google-closure-compiler-js": "^20170409.0.0",
```

Comment:
Is it used currently, or was it added for future experimentation? It not used let's remove it.